sdk/script/management-sdk.ts

```diff
@@ -257,8 +251,16 @@ class AccountManager {
         });
     }
 
-    public promote(appName: string, sourceDeploymentName: string, destDeploymentName: string): Promise<void> {
-        return this.post(urlEncode `/apps/${appName}/deployments/${sourceDeploymentName}/promote/${destDeploymentName}`, /*requestBody=*/ null, /*expectResponseBody=*/ false)
+    public patchRelease(appName: string, deploymentName: string, label: string, updateMetadata: PackageInfo): Promise<void> {
```

Comment:
@dtivel - I have sent out a PR #166 for the corresponding changes. Please take a look.